python/lsst/__init__.py

```diff
@@ -1,3 +1,2 @@
 import pkgutil
-import lsstimport
```

Comment:
I was curious to see that this was still here, and maybe you already knew this, but `lsstimport` is apparently totally empty now, and can be removed everywhere.

I'm also curious to see that the `pkgutil` bit below that hasn't changed - did we revert using namespace packages at some point?  If so, we could probably go simplify the daf_butler mypy build.  If not, is this relic a time bomb - I thought namespace packages and the `pkgutil` snippet weren't compatible, so we might be getting by on the fact that the right one tends to appear first in `sys.path`.

Reply:
I was the person that turned lsstimport into a stub when I realized we only needed it in one place (afw I think).  Yes, I've given up on the namespace packages -- too much still goes wrong if you remove it. We get by because we fill PYTHONPATH up with all sorts of different options for that directory and something usually wins. I think mypy was trying to tell us something so probably best to put it back.